recipes/lmfit/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.9.3" %}
+
+package:
+  name: lmfit
+  version: {{ version }}
+
+source:
+  url: https://github.com/lmfit/lmfit-py/archive/{{ version }}.tar.gz
+  fn: lmfit-{{ version }}.tar.gz
+  sha256: e15c629b30cd70da525db6e66e3532d923a95e7a5f183906c83badf22c6be70c
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+
+  run:
+    - emcee
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - pandas
+    - python
+    - scipy
+
+test:
+  requires:
+    - nose
+  commands:
+    - nosetests -sv {{ environ.SRC_DIR }}
+  imports:
+    - lmfit
+
+about:
+  home: http://lmfit.github.io/lmfit-py/
+  license: BSD
```

Comment:
Alright, let's resolve this. Some options. Either seem fine to me.

1. `BSD-like`. [Example]( https://github.com/conda-forge/pyopengl-feedstock/blob/9dbfdeba92891e1b73112786d44c76013ae6f534/recipe/meta.yaml#L30 ).
2. Call it the `lmfit` license. [Example]( https://github.com/conda-forge/hdf5-feedstock/blob/38ee94d15eff502b4c387def4ae87c759978aee3/recipe/meta.yaml#L93-L94 ).